src/seedu/addressbook/commands/ListByCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class ListByCommand extends Command {
+
+    public static final String COMMAND_WORD = "listBy";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Displays all persons in the address book as a list sorted by a user specified field.\n\t"
+            + "Parameters: \"name\", \"phone\", \"email\" or \"address\".\n\t" + "Example: " + COMMAND_WORD + " name";
```

Comment:
Add a new line in front of "Example:". This follows the style used by the other commands - the new lines in code follow the new lines in the string.